test/unified_format.py

```diff
@@ -992,6 +998,8 @@ def maybe_skip_test(self, spec):
             self.skipTest("Implement PYTHON-1894")
         if "timeoutMS applied to entire download" in spec["description"]:
             self.skipTest("PyMongo's open_download_stream does not cap the stream's lifetime")
+        if "command log messages include server connection id" == spec["description"]:
+            self.skipTest("TODO: Implement https://jira.mongodb.org/browse/PYTHON-2822")
```

Comment:
This is fixed now. Could you open a new ticket unless you plan to address in this PR?